db/seeds.rb

```diff
@@ -33,3 +37,102 @@
       hash.reject { |key, _| key =~ /ext|public_service/ }
     )
 end
+
+WorkingStatus.create(value: 'Sans activité')
```

Comment:
This is good for development, but How are we going to populate production instances without messing up with real and existing data?